### ISSUE TITLE
Harmony payments are failing 

### ISSUE THREAD

**Issue by GTChase** [gitcoinco/web#10428]
**Overview**
Harmony has stopped supporting the harmony chrome connector (harmony wallet  extension). Need to properly implement the ethereum SDK to talk to harmony blockchain using metamask. 

**Steps To Reproduce**
1. Open bounty being paid out on Harmony
2. Select ONE as payout token
3. Have someone submit work to bounty
4. Payout contributor

**Observed  behavior**
Payout tries to happen using the harmony wallet extension and that is no longer supported.. Causing gas fees to be too low to successfully go through

**Expected behavior**
Please update the gitcoin website to use the standard ethereum sdk to talk to harmony blockchain using metamask.
https://docs.harmony.one/home/general/wallets/browser-extensions-wallets/metamask-wallet

**Screenshots**
![image](https://user-images.githubusercontent.com/95642312/160506445-6c37d44c-2975-40b3-9c35-17d94ea9009e.png)

**Additional context**
**Closed bounty that was having issues:** https://gitcoin.co/issue/harmony-one/bounties/111/100028644
**Discord discussions - see videos:** https://discord.com/channels/562828676480237578/920714502650077264/956106612328783882

**Comment by nutrina** [comment]
The current implementation for ETH works as indicated in the snapshot below.
If you agree I will try to implement the same process. However, I think that we need to implement a validation step, in order to check that the user is on the correct chain. In the screenshot below, as you see I am prompted to confirm a transaction on Polygon, even though the selected chain in the bounty was ETH. 
I think we should help the user prevent making such mistakes, and only allow the final step if he is on the correct chain.  

<img width="1204" alt="image" src="https://user-images.githubusercontent.com/8137830/160571464-c9284151-5139-4a85-a05e-ef39506c3c05.png">

**Comment by GTChase** [comment]
Hoping for Morten to chime in on the first question..

As for the second question, i do agree thats an issue, but lets hold off on fixing this right now with this issue. Id like to move our priority back to finishing the bounty creation workflow.  i can create a new issue to hold the fix you mentioned above and continue to prioritize it against other items.

**Comment by mopdo** [comment]
Yes to both from my side here as well Gerald, good thinking! It's up to Chase decide the prioritization though, for now and for this issue they would hit a barrier when they have no "ONE" tokens on any other (in this case "wrong") network though so there's a failsafe in the end. 

Cheers guys!

**Comment by GTChase** [comment]
Gerald walked me through the proposed workflow for Harmony using Meta and all looked good on my end as well. 

Double sign off, carry forward!

**Comment by nutrina** [comment]
Another point to think about: do other web3 wallets besides Metamask support Harmony?
We generally allow the usage of multiple wallets through Web3Modal, so we might keep in mind to check that all options for the users there are suited for Harmony as well.

**Comment by nutrina** [comment]
Also, the same adjustments should be made for grants too.

**Comment by GTChase** [comment]
> Another point to think about: do other web3 wallets besides Metamask support Harmony? We generally allow the usage of multiple wallets through Web3Modal, so we might keep in mind to check that all options for the users there are suited for Harmony as well.

@nutrina i am not fully sure, but this was the request from Harmony's VP of engineering that i received: Please update the gitcoin website to use the standard ethereum sdk to talk to harmony blockchain using metamask.

So it feels like starting with metamask is the right thing to do.

**Comment by GTChase** [comment]
> Also, the same adjustments should be made for grants too.

i will work with Nate on getting a ticket for their side of the world spun up to make sure this is ready to go for grants.
